lib/index.js

```diff
@@ -155,13 +157,45 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         }).type('application/ld+json').send(inbox));
   });
 
+  // get inbox info, including list of messages
```

Comment:
Is this comment right? Are lists of messages returned? If so, is that what we want? I would think not.